### DISCUSSION TITLE
New branch protection rule for 'develop' branch

### DISCUSSION THREAD

**Discussion by kwwall** [ESAPI/esapi-java-legacy#766] [Announcements]
Just an FYI - I've added "branch protection rule" for the 'develop' branch (our default branch) that requires commits to be signed and at least one code reviewer before a PR can be merged. Please let me know if there is any trouble.

**Comment by xeno6696** [comment]
Roger that.  Sent from my iPhoneOn Jan 7, 2023, at 15:46, Kevin W. Wall ***@***.***> wrote:﻿
Just an FYI - I've added "branch protection rule" for the 'develop' branch (our default branch) that requires commits to be signed and at least one code reviewer before a PR can be merged. Please let me know if there is any trouble.
